src/megatron/energon/tools/preview.py

```diff
@@ -96,27 +102,37 @@ def pprint(idx: int, sample: Sample):
     "--split-parts", default="train,val,test", help="The splits to verify", show_default=True
 )
 @click.option(
-    "--config", default="dataset.yaml", help="Dataset config file name", show_default=True
+    "--dataset-config", default="dataset.yaml", help="Dataset config file name", show_default=True
 )
-def command(path: EPath, split_parts: str, config: str):
+def command(path: EPath, split_parts: str, dataset_config: str):
     """Views the contents of a dataset on the console."""
 
     path = path.absolute()
 
+    worker_config = WorkerConfig(rank=0, world_size=1, num_workers=0)
+
     for split_part in split_parts.split(","):
-        dataset = load_config(
-            EPath(path) / MAIN_FOLDER_NAME / config,
-            default_kwargs={
-                "path": path,
-                "split_part": split_part,
-                "training": False,
-            },
-        )
+        try:
+            dataset = load_config(
+                EPath(path) / MAIN_FOLDER_NAME / dataset_config,
+                default_kwargs=dict(
+                    path=path,
+                    split_part=split_part,
+                    training=False,
+                    worker_config=worker_config,
+                ),
+                default_type=StandardWebdatasetFactory,
```

Comment:
Why is this non-optional now? (default_type)

Reply:
Because it was wrong before :D All calls now have a default type anyways